config/routes.rb

```diff
@@ -9,6 +9,7 @@
       resources :courses,     only: [:index]
       resources :sections,    only: [:index]
       resources :schedules,   only: [:index]
+	  resources :conflicts,   only: [:index]
```

Comment:
There does not appear to be a 'conflicts' request anywhere. Removed this conflicts route.